docs/admin/ServerSideKeygen.adoc

```diff
@@ -0,0 +1,157 @@
+= ServerSide Key Generation for EE
+Author <cfu@redhat.com>
```

Comment:
I think this should be: `Christina Fu <cfu@redhat.com>`.

According to [the doc](https://asciidoctor.org/docs/user-manual/#author-and-email) the format seems to be correct, but for some reason it doesn't appear in GitHub:
https://github.com/ladycfu/pki/blob/Doc-bug1629025-SeverSide-Keygen/docs/admin/ServerSideKeygen.adoc

Reply:
Oh, I actually thought it was just meta data not meant to be displayed, which is what I prefer anyway.  But you did point out something I misunderstood.  I did not realize that the word "Author" was supposed to be replaced with my name ;-).  I'll replace that.